abs.yaml

```diff
@@ -0,0 +1,9 @@
+build_env_vars:
+  ANACONDA_ROCKET_ENABLE_PY313 : yes
```

Comment:
@cbouss can we try the following and see if that unblocks the issue?

```suggestion
  ANACONDA_ROCKET_ENABLE_PY313: yes
  _CONDA_BUILD_ISOLATED_ACTIVATION: 1
```

Reply:
That worked, thanks!